pyqmc/dmc.py

```diff
@@ -201,6 +205,36 @@ def dmc_propagate(wf,configs,weights,tstep,nsteps=5,accumulators=None,ekey=('ene
         df.append(avg)
     return df, configs, weights
     
+def stabilize_weights(weights, elocnew, elocold, eref, erefsigma, branchcut_start, branchcut_stop):
```

Comment:
Same here, instead of erefsigma and branchcut_start/branchcut_stop, just stop/start.

The name seems a bit awkward as well. You are actually returning the limited timestep; the weights are not getting stabilized here.